common/apitypes/pooll2txapi.go

```diff
@@ -0,0 +1,147 @@
+package apitypes
+
+import (
+	"encoding/json"
+	"time"
+
+	ethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/hermeznetwork/hermez-node/common"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+)
+
+// PoolL2TxAPI represents a L2 Tx pool with extra metadata used by the API
+type PoolL2TxAPI struct {
```

Comment:
Unfortunately, the `json` is missing here. The SDK and other 3rd-party softwares would need the json tags in here to Unmarshall when it receives the return the json string of the API REST call. Could you please add them here?